src/integrationTest/java/uk/gov/hmcts/reform/pip/subscription/management/controllers/SubscriptionControllerTests.java

```diff
@@ -727,6 +728,65 @@ void testBuildDeletedArtefactSubscribersReturnsAccepted() throws Exception {
         );
     }
 
+    @Test
+    void testBuildBulkDeletedSubscribersReturnsAccepted() throws Exception {
```

Comment:
We should also test that the subscription has been deleted here by calling the get endpoint to make sure it's gone

Reply:
done